### PR TITLE
AWS Lambda enhancement 6 commands

### PR DESCRIPTION
**Pull request by shmuel44**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link](https://jira-hq.paloaltonetworks.local/browse/CIAC-6665) to the issue

## Description
Added a 6 new commands and some improvement,
Changed the integration path from aws-lambda to aws_lambda and the name of the  python file.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 1]
```diff
@@ -0,0 +1,1230 @@
...
+      description: You can set this optional parameter to Tail in the request only if you specify the InvocationType parameter with value RequestResponse . In this case, AWS Lambda returns the base64-encoded last 4 KB of log data produced by your Lambda function in the x-amz-log-result header.
+      isArray: false
+      name: logType
+      predefined:
+      - None
+      - Tail
+      required: false
+      secret: false
+    - default: false
+      description: 'Using the ClientContext you can pass client-specific information to the Lambda function you are invoking. '
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:470]:
```suggestion
      description: Using the ClientContext you can pass client-specific information to the Lambda function you are invoking.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 2]
```diff
@@ -0,0 +1,1230 @@
...
+      secret: false
+    - default: false
+      description: Specify a version or alias to invoke a published version of the function.
+      isArray: false
+      name: qualifier
+      required: false
+      secret: false
+    - default: false
+      auto: PREDEFINED
+      description: The AWS Region, if not specified the default region will be used
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:489]:
```suggestion
      description: The AWS Region. If not specified, the default region will be used.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 3]
```diff
@@ -0,0 +1,1230 @@
...
+      - ap-southeast-1
+      - ap-southeast-2
+      - ap-south-1
+      - sa-east-1
+      - eu-north-1
+      - eu-west-3
+      required: false
+      secret: false
+    - default: false
+      description: The Amazon Resource Name (ARN) of the role to assume
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:512]:
```suggestion
      description: The Amazon Resource Name (ARN) of the role to assume.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 4]
```diff
@@ -0,0 +1,1230 @@
...
+      required: false
+      secret: false
+    - default: false
+      description: The duration, in seconds, of the role session. The value can range from 900 seconds (15 minutes) up to the maximum session duration setting for the role.
+      isArray: false
+      name: roleSessionDuration
+      required: false
+      secret: false
+    - default: false
+      description: The maximum retry attempts when connection or throttling errors are encountered. Set to 0 to disable retries. If not, specified will use the instances configured default timeout.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:530]:
```suggestion
      description: The maximum retry attempts when connection or throttling errors are encountered. Set to 0 to disable retries. If not specified, will use the instances configured default timeout.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 5]
```diff
@@ -0,0 +1,1230 @@
...
+      required: false
+      secret: false
+    - default: false
+      description: The time in seconds till a timeout exception is reached. You can specify just the read timeout (for example 60) or also the connect timeout followed after a comma (for example 60,10). If not specified, will use the instances configured default timeout.
+      isArray: false
+      name: timeout
+      required: false
+      secret: false
+    deprecated: false
+    description: Invokes a Lambda function. Specify just a function name to invoke the latest version of the function. To invoke a published version, use the Qualifier parameter to specify a version or alias .  If you use the RequestResponse (synchronous) invocation option, note that the function may be invoked multiple times if a timeout is reached. For functions with a long timeout, your client may be disconnected during synchronous invocation while it waits for a response. Use the "timeout" and "retries" args to control this behavior. If you use the Event (asynchronous) invocation option, the function will be invoked at least once in response to an event and the function must be idempotent to handle this.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:542]:
```suggestion
    description: Invokes a Lambda function. Specify just a function name to invoke the latest version of the function. To invoke a published version, use the Qualifier parameter to specify a version or alias.  If you use the RequestResponse (synchronous) invocation option, note that the function may be invoked multiple times if a timeout is reached. For functions with a long timeout, your client may be disconnected during synchronous invocation while it waits for a response. Use the "timeout" and "retries" arguments to control this behavior. If you use the Event (asynchronous) invocation option, the function will be invoked at least once in response to an event and the function must be idempotent to handle this.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 6]
```diff
@@ -0,0 +1,1230 @@
...
+    execution: true
+    name: aws-lambda-invoke
+    outputs:
+    - contextPath: AWS.Lambda.InvokedFunctions.FunctionName
+      description: The name of the Lambda function.
+      type: string
+    - contextPath: AWS.Lambda.InvokedFunctions.FunctionError
+      description: Indicates whether an error occurred while executing the Lambda function. If an error occurred this field will have one of two values; Handled or Unhandled . Handled errors are errors that are reported by the function while the Unhandled errors are those detected and reported by AWS Lambda. Unhandled errors include out of memory errors and function timeouts.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:550]:
```suggestion
      description: Indicates whether an error occurred while executing the Lambda function. If an error occurred this field will have one of two values; Handled or Unhandled. Handled errors are errors that are reported by the function while the Unhandled errors are those detected and reported by AWS Lambda. Unhandled errors include out of memory errors and function timeouts.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 7]
```diff
@@ -0,0 +1,1230 @@
...
+      type: string
+    - contextPath: AWS.Lambda.InvokedFunctions.LogResult
+      description: logs for the Lambda function invocation. This is present only if the invocation type is RequestResponse and the logs were requested.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:553]:
```suggestion
      description: Logs for the Lambda function invocation. This is present only if the invocation type is RequestResponse and the logs were requested.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 8]
```diff
@@ -0,0 +1,1230 @@
...
+      type: string
+    - contextPath: AWS.Lambda.InvokedFunctions.Payload
+      description: It is the JSON representation of the object returned by the Lambda function. This is present only if the invocation type is RequestResponse.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:556]:
```suggestion
      description: The JSON representation of the object returned by the Lambda function. This is present only if the invocation type is RequestResponse.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 9]
```diff
@@ -0,0 +1,1230 @@
...
+      type: string
+    - contextPath: AWS.Lambda.InvokedFunctions.ExecutedVersion
+      description: The function version that has been executed. This value is returned only if the invocation type is RequestResponse.
+      type: string
+    - contextPath: AWS.Lambda.InvokedFunctions.Region
+      description: The AWS Region.
+      type: string
+  - arguments:
+    - default: false
+      description: The AWS Region, if not specified the default region will be used
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:566]:
```suggestion
      description: The AWS Region. If not specified, the default region will be used.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 10]
```diff
@@ -0,0 +1,1230 @@
...
+      - ap-southeast-1
+      - ap-southeast-2
+      - ap-south-1
+      - sa-east-1
+      - eu-north-1
+      - eu-west-3
+      required: false
+      secret: false
+    - default: false
+      description: The Amazon Resource Name (ARN) of the role to assume
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:590]:
```suggestion
      description: The Amazon Resource Name (ARN) of the role to assume.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 11]
```diff
@@ -0,0 +1,1230 @@
...
+      description: The maximum size of your function's code and layers when they're extracted.
+      type: number
+    - contextPath: AWS.Lambda.AccountLimit.CodeSizeZipped
+      description: The maximum size of a deployment package when it's uploaded directly to AWS Lambda. Use Amazon S3 for larger files.
+      type: number
+    - contextPath: AWS.Lambda.AccountLimit.ConcurrentExecutions
+      description: The maximum number of simultaneous function executions.
+      type: number
+    - contextPath: AWS.Lambda.AccountLimit.UnreservedConcurrentExecutions
+      description: The maximum number of simultaneous function executions, minus the capacity that's reserved for individual functions with PutFunctionConcurrency .
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:625]:
```suggestion
      description: The maximum number of simultaneous function executions, minus the capacity that's reserved for individual functions with PutFunctionConcurrency.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 12]
```diff
@@ -0,0 +1,1230 @@
...
+      type: number
+    - contextPath: AWS.Lambda.AccountUsage.TotalCodeSize
+      description: The amount of storage space, in bytes, that's being used by deployment packages and layer archives.
+      type: number
+    - contextPath: AWS.Lambda.AccountUsage. FunctionCount
+      description: The number of Lambda functions.
+      type: number
+  - arguments:
+    - default: false
+      description: The AWS Region, if not specified the default region will be used
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:635]:
```suggestion
      description: The AWS Region. If not specified, the default region will be used.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 13]
```diff
@@ -0,0 +1,1230 @@
...
+      - ap-southeast-1
+      - ap-southeast-2
+      - ap-south-1
+      - sa-east-1
+      - eu-north-1
+      - eu-west-3
+      required: false
+      secret: false
+    - default: false
+      description: The Amazon Resource Name (ARN) of the role to assume
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:659]:
```suggestion
      description: The Amazon Resource Name (ARN) of the role to assume.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 14]
```diff
@@ -0,0 +1,1230 @@
...
+      type: String
+    - contextPath: AWS.Lambda.Policy.Statement.Resource
+      description: The resource ARN specified in the AWS Lambda policy statement.
+      type: String
+    - contextPath: AWS.Lambda.RevisionId
+      description: A unique identifier for the current revision of the policy.
+      type: String
+  - arguments:
+    - default: false
+      description: The AWS Region, if not specified the default region will be used
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:709]:
```suggestion
      description: The AWS Region. If not specified, the default region will be used.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 15]
```diff
@@ -0,0 +1,1230 @@
...
+      - ap-southeast-1
+      - ap-southeast-2
+      - ap-south-1
+      - sa-east-1
+      - eu-north-1
+      - eu-west-3
+      required: false
+      secret: false
+    - default: false
+      description: The Amazon Resource Name (ARN) of the role to assume
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:733]:
```suggestion
      description: The Amazon Resource Name (ARN) of the role to assume.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 16]
```diff
@@ -0,0 +1,1230 @@
...
+      type: String
+    - contextPath: AWS.Lambda.Versions.ImageConfigResponse.ImageConfigError.Type
+      description: The error type for image configuration.
+      type: String
+    - contextPath: AWS.Lambda.Versions.ImageConfigResponse.ImageConfig
+      description: The image configuration values.
+      type: Object
+  - arguments:
+    - default: false
+      description: The AWS Region, if not specified the default region will be used
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:873]:
```suggestion
      description: The AWS Region. If not specified, the default region will be used.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 17]
```diff
@@ -0,0 +1,1230 @@
...
+      - ap-southeast-1
+      - ap-southeast-2
+      - ap-south-1
+      - sa-east-1
+      - eu-north-1
+      - eu-west-3
+      required: false
+      secret: false
+    - default: false
+      description: The Amazon Resource Name (ARN) of the role to assume
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:897]:
```suggestion
      description: The Amazon Resource Name (ARN) of the role to assume.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 18]
```diff
@@ -0,0 +1,1230 @@
...
+      type: List
+    - contextPath: AWS.Lambda.FunctionURLConfig.Cors.AllowMethods
+      description: The HTTP methods that are allowed when calling the function URL. For example GET, POST, DELETE, or the wildcard character ( *).
+      type: List of strings
+    - contextPath: AWS.Lambda.FunctionURLConfig.Cors.AllowOrigins
+      description: The origins that can access the function URL.You can list any number of specific origins, separated by a comma. For example https://www.example.com, http://localhost:8080. Alternatively, you can grant access to all origins using the wildcard character ( *).
+      type: List of strings
+    - contextPath: AWS.Lambda.FunctionURLConfig.Cors.ExposeHeaders
+      description: The HTTP headers in the function response that you want to expose to origins that call the function URL. For example Date, Keep-Alive, X-Custom-Header.
+      type: List of strings
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:947]: Is this type correct? I've never seen "List of strings" as a type.

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 19]
```diff
@@ -0,0 +1,1230 @@
...
+      type: String
+    - contextPath: AWS.Lambda.FunctionURLConfig.LastModifiedTime
+      description: When the function URL configuration was last updated, in ISO-8601 format (YYYY-MM-DDThh:mm:ss.sTZD).
+      type: String
+    - contextPath: AWS.Lambda.FunctionURLConfig.InvokeMode
+      description: "Use one of the following options: BUFFERED – This is the default option. Lambda invokes your function using the Invoke API operation. Invocation results are available when the payload is complete. The maximum payload size is 6 MB. RESPONSE_STREAM – Your function streams payload results as they become available. Lambda invokes your function using the InvokeWithResponseStream API operation. The maximum response payload size is 20 MB, however, you can request a quota increase."
+      type: String
+  - arguments:
+    - default: false
+      description: The AWS Region, if not specified the default region will be used
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:962]:
```suggestion
      description: The AWS Region. If not specified, the default region will be used.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 20]
```diff
@@ -0,0 +1,1230 @@
...
+      - ap-southeast-1
+      - ap-southeast-2
+      - ap-south-1
+      - sa-east-1
+      - eu-north-1
+      - eu-west-3
+      required: false
+      secret: false
+    - default: false
+      description: The Amazon Resource Name (ARN) of the role to assume
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:986]:
```suggestion
      description: The Amazon Resource Name (ARN) of the role to assume.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 21]
```diff
@@ -0,0 +1,1230 @@
...
+      type: String
+    - contextPath: AWS.Lambda.FunctionConfig.ImageConfigResponse.ImageConfigError.Type
+      description: The error type for image configuration.
+      type: String
+    - contextPath: AWS.Lambda.FunctionConfig.ImageConfigResponse.ImageConfig
+      description: The image configuration values.
+      type: Object
+  - arguments:
+    - default: false
+      description: The AWS Region, if not specified the default region will be used
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:1120]:
```suggestion
      description: The AWS Region. If not specified, the default region will be used.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 22]
```diff
@@ -0,0 +1,1230 @@
...
+      - ap-southeast-1
+      - ap-southeast-2
+      - ap-south-1
+      - sa-east-1
+      - eu-north-1
+      - eu-west-3
+      required: false
+      secret: false
+    - default: false
+      description: The Amazon Resource Name (ARN) of the role to assume
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:1144]:
```suggestion
      description: The Amazon Resource Name (ARN) of the role to assume.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 23]
```diff
@@ -0,0 +1,1230 @@
...
+      required: true
+      name: functionName
+    - default: false
+      description: Specify a version or alias to get details about a published version of the function.
+      name: qualifier
+    description: Deletes a Lambda function URL. When you delete a function URL, you can’t recover it. Creating a new function URL results in a different URL address.
+    name: aws-lambda-delete-function-url-config
+  - arguments:
+    - default: false
+      description: The AWS Region, if not specified the default region will be used
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:1172]:
```suggestion
      description: The AWS Region. If not specified, the default region will be used.
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml [hunk 24]
```diff
@@ -0,0 +1,1230 @@
...
+      - ap-southeast-1
+      - ap-southeast-2
+      - ap-south-1
+      - sa-east-1
+      - eu-north-1
+      - eu-west-3
+      required: false
+      secret: false
+    - default: false
+      description: The Amazon Resource Name (ARN) of the role to assume
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.yml:1196]:
```suggestion
      description: The Amazon Resource Name (ARN) of the role to assume.
```

**Comment by ShirleyDenkberg** [comment]
@shmuel44 After implementing my yml comments, please regenerate the README.
@israelpoli @dorschw Doc review completed.

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.py [hunk 25]
```diff
@@ -0,0 +1,596 @@
...
+                     - 'qualifier' (str, optional): The qualifier of the function.
+
+        aws_client (boto3 client): The AWS client used to delete the function.
+
+    Returns:
+        CommandResults: An object containing the result of the deletion operation as a readable output in Markdown format.
+    """
+    kwargs = {'FunctionName': args['functionName']}
+    if qualifier := args.get('qualifier'):
+        kwargs.update({'qualifier': qualifier})
```
Comment (dorschw) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.py:520]:
```suggestion
        kwargs['qualifier'] = qualifier
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.py [hunk 26]
```diff
@@ -0,0 +1,596 @@
...
+                return_results(list_versions_by_function_command(args, aws_client))
+            case 'aws-lambda-get-function-url-config':
+                return_results(get_function_url_config_command(args, aws_client))
+            case 'aws-lambda-get-function-configuration':
+                return_results(get_function_configuration_command(args, aws_client))
+            case 'aws-lambda-delete-function-url-config':
+                return_results(delete_function_url_config_command(args, aws_client))
+            case 'aws-lambda-delete-function':
+                return_results(delete_function_command(args, aws_client))
+    except Exception as e:
```
Comment (dorschw) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.py:589]: please add 
```python
case _:
    raise NotImplementedError(f"Command {command} is not implemented")
```

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda_test.py [hunk 27]
```diff
@@ -0,0 +1,460 @@
...
+    client = MockClient()
+    mocker.patch.object(client, "list_versions_by_function", return_value=test_data)
+
+    res = list_versions_by_function_command(
+        args={"functionName": "test"}, aws_client=client
+    )
+    if test_data.get("NextMarker") and res.readable_output:
+        assert (
+            "To get the next version run the command with the Marker argument with the value: test"
+        ) in res.readable_output
```
Comment (dorschw) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda_test.py:276]:
```suggestion
        assert (test_data.get("NextMarker") and res.readable_output) == (
            "To get the next version run the command with the Marker argument with the value: test"
        ) in res.readable_output
```

File: Packs/AWS-Lambda/TestPlaybooks/playbook-AWS-Lambda-Test-(Read-Only).yml
Comment (dorschw) [inline Packs/AWS-Lambda/TestPlaybooks/playbook-AWS-Lambda-Test-(Read-Only).yml:1]: please run `format` on this file

File: Packs/AWS-Lambda/TestPlaybooks/playbook-AWS-Lambda-Test-(Read-Only).yml [hunk 28]
```diff
@@ -101,50 +108,36 @@ tasks:
     quietmode: 0
     reputationcalc: 1
     scriptarguments:
-      clientContext: {}
       functionName:
-        simple: test_echo
-      invocationType: {}
+        simple: XSIAM-Dev
```
Comment (dorschw) [inline Packs/AWS-Lambda/TestPlaybooks/playbook-AWS-Lambda-Test-(Read-Only).yml:112]: ?

**Review by dorschw — COMMENTED** [review]
Very nice

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.py [hunk 29]
```diff
@@ -0,0 +1,596 @@
...
+    ec = {'AWS.Lambda.Functions(val.FunctionArn === obj.FunctionArn)': raw}
+    human_readable = tableToMarkdown('AWS Lambda Functions', data)
+    return_outputs(human_readable, ec)
+
+
+def list_functions(aws_client):
+
+    obj = vars(aws_client._client_config)
+    data = []
+    output = []
```
[the remark this answers is not in the document]
Comment (shmuel44) [reply Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.py:167]: I don't know, old code

[the remark this answers is not in the document]
Comment (shmuel44) [reply Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.py:167]: old code

**Review by dorschw — APPROVED** [review]

**Pull request merged**